Assets/Scripts/TES/MorrowindEngine.cs

```diff
@@ -854,7 +860,7 @@ private GameObject CreatePlayerCamera(Vector3 position)
 		{
 			var camera = GameObjectUtils.CreateMainCamera(position, Quaternion.identity);
 			camera.GetComponent<Camera>().cullingMask = ~(1 << markerLayer);
-
+			Camera.main.renderingPath = RenderingPath.DeferredShading;
```

Comment:
Of course! i had actually meant to make it a toggle on the TESUnity object but hadnt gotten around to it yet. that would definitely be optimal.